README.md

```diff
@@ -21,3 +19,28 @@ that will suggest people matching an entered text.
 - Implement a solution following the [React task guideline](https://github.com/mate-academy/react_task-guideline#react-tasks-guideline).
 - Use the [React TypeScript cheat sheet](https://mate-academy.github.io/fe-program/js/extra/react-typescript).
 - Replace `<your_account>` with your Github username in the [DEMO LINK](https://<your_account>.github.io/react_autocomplete/) and add it to the PR description.
+- Don't remove the `data-qa` attributes. It is required for tests.
+
+## Troubleshooting
+
+If you have got an error during `npm install` like this (when it tries to install `node-sass`):
+
+```bash
+npm ERR! code ELIFECYCLE
+npm ERR! errno 1
+npm ERR! node-sass@6.0.1 postinstall: `node scripts/build.js`
+npm ERR! Exit status 1
+npm ERR!
+npm ERR! Failed at the node-sass@6.0.1 postinstall script.
+npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
+```
+
+Then you probably have an incompatible version of Python installed.
+You can check it by running `python --version` command.
```

Comment:
We have never asked our students to install `python` or check its version in the other React tasks using `scss`

So the correct package to install now is `sass` but not `node-sass` 
https://mate.academy/learn/react/introduction-react?section=video&videoId=5262

Reply:
@sergii-nosachenko But `package.json` in this task has `node-sass` dependency.